master/usage/configuration/ip-in-ip.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Configuring IP-in-IP
+---
+
+Calico can be configured to use IP-in-IP encapsulation for inter-host inter-workload
+traffic.  The option is configured on the [IP pool resource]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/ippool),
+and when enabled Calico will use IP-in-IP encapsulation for packets routed *to*
+workload IPs in the range of the IP pool.
+
+An optional `mode` field toggles when IP-in-IP is used:
+-  A value of `always` is used when IP-in-IP is always required.  This is the default
+   value used if the `mode` field is omitted in the configuration.
+-  A value of `cross-subnet` is used when IP-in-IP is only required when routing 
+   between hosts on different subnets.
+
+> **NOTE**
+>
+> Routing between Workloads and Host Endpoints running Calico is asymmetric when
+> IPIP is enabled, this causes traffic to be discarded between them.  To resolve 
+> this, enable nat-outgoing or use the IPIP tunnel address on the host
+> for the Host Endpoint IP address.
+
+### Configuring cross-subnet IP-in-IP
+
+The following `calicoctl` example will create or modify an IPv4 pool with 
+CIDR 192.168.0.0/16 using IPIP mode `cross-subnet`.
+
+```
+$ calicoctl apply -f - << EOF
+apiVersion: v1
+kind: ipPool
+metadata:
+  cidr: 192.168.0.0/16
+spec:
+  ipip:
+    enabled: true
+    mode: cross-subnet
+EOF
+```
+
+> **Note** 
+>
+> The `cross-subnet` mode option requires each Calico node to be configured
+> with the IP address and subnet of the host.  However, the subnet configuration
+> was only introduced in Calico v2.1.  If any nodes in your deployment were originally
+> created with an older version of Calico, or if you if you are unsure whether 
+> your deployment is configured correctly, follow the steps in 
+> [Upgrading from pre-v2.1](#upgrading-from-pre-v21) before enabling "cross-subnet" IPIP.
+
+#### Upgrading from pre-v2.1
+
+If you are planning to use cross-subnet IPIP, your entire deployment must be running with
+Calico v2.1 or higher.  See [releases page]({{site.baseurl}}/{{page.version}}/releases) 
+for details on the component versions for each release.
+
+Upgrade your deployment to use the latest Calico versions - the process for this
+will be dependent on your orchestration system (if using one). 
+
+If you previously had calico/node instances that were deployed prior to v2.1, the
```

Comment:
Can you go a little deeper into the exact reason? Which is:

>Since Calico v2.1, Calico detects the subnet on the interface at startup and stores that information in etcd. If you're upgrading from v2.0, you'll need to perform this step manually.